Dockerfile.worker

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7-alpine
+
+RUN pip install pipenv
+
+COPY Pipfile Pipfile
+COPY Pipfile.lock Pipfile.lock
+
+RUN apk update && apk add --no-cache gcc libffi-dev openssl-dev g++ postgresql-dev make
+
+RUN pipenv install --system --deploy --ignore-pipfile
+
+RUN apk del libffi-dev g++ make
```

Comment:
rebuilds will be made a great deal faster if you do the slow, steps that don't change often first.

Here you'll need to reinstall all these packages every time the `Pipfile` changes.